inc/commonitilobject.class.php

```diff
@@ -6263,66 +6222,126 @@ static function showShort($id, $options = []) {
+                  $items[$plan['id']] = $plan['id'];
+                  $planned_infos .= sprintf(__('From %s').
+                                             ($p['output_type'] == Search::HTML_OUTPUT?'<br>':''),
+                                          Html::convDateTime($plan['begin']));
+                  $planned_infos .= sprintf(__('To %s').
+                                             ($p['output_type'] == Search::HTML_OUTPUT?'<br>':''),
+                                          Html::convDateTime($plan['end']));
+                  if ($plan['users_id_tech']) {
+                     $planned_infos .= sprintf(__('By %s').
+                                                ($p['output_type'] == Search::HTML_OUTPUT?'<br>':''),
+                                             getUserName($plan['users_id_tech']));
+                  }
+                  $planned_infos .= "<br>";
                }
-               $planned_infos .= "<br>";
+
             }
 
-         }
+            $tenth_column = count($items);
+            if ($tenth_column) {
+               $tenth_column = "<span class='pointer'
+                                 id='".$item->getType().$item->fields["id"]."planning$rand'>".
+                                 $tenth_column.'</span>';
+               $tenth_column = sprintf(__('%1$s %2$s'), $tenth_column,
+                                       Html::showToolTip($planned_infos,
+                                                         ['display' => false,
+                                                               'applyto' => $item->getType().
+                                                                              $item->fields["id"].
+                                                                              "planning".$rand]));
+            }
 
-         $tenth_column = count($items);
-         if ($tenth_column) {
-            $tenth_column = "<span class='pointer'
-                              id='".$item->getType().$item->fields["id"]."planning$rand'>".
-                              $tenth_column.'</span>';
-            $tenth_column = sprintf(__('%1$s %2$s'), $tenth_column,
-                                    Html::showToolTip($planned_infos,
-                                                      ['display' => false,
-                                                            'applyto' => $item->getType().
-                                                                           $item->fields["id"].
-                                                                           "planning".$rand]));
+            echo Search::showItem($p['output_type'], $tenth_column, $item_num, $p['row_num'],
+                                 $align_desc." width='150'");
+         } else {
+            echo Search::showItem($p['output_type'], $name_column, $item_num, $p['row_num'], $align_desc." width='200'");
+            $takeintoaccountdelay_column = Html::timestampToString($item->fields['takeintoaccount_delay_stat']);
```

Comment:
It may happen that this value is not yet set, it should be handled.

Reply:
Isn't it always set here ?
![image](https://user-images.githubusercontent.com/42734840/114516467-b9dae280-9c3d-11eb-802e-5742d35179f0.png)